modules/shared/components/molecules/CreatePostHeader/CreatePostHeader.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import type { FC, ReactElement } from 'react';
+import type { ICreatePostHeader } from './ICreatePostHeader';
+import Avatar from '../../atoms/Avatar/Avatar';
+import TabGroup from '../TabGroup/TabGroup';
+
+const CreatePostHeader: FC<ICreatePostHeader.IProps> = (
+  props,
+): ReactElement => {
+  const { tabsData, checkedValue, changeValHandler, profilePic } = props;
+
+  const avatarVariant = profilePic ? 'filled' : 'notFilled';
```

Comment:
typeScript is smart enough to infer the types in the context, but the more explicit the better.

Reply:
i thought about it first but when i hovered on the constant i saw that it's already declaring the type

![Screenshot from 2021-06-12 14-36-20](https://user-images.githubusercontent.com/52901348/121776205-da66e080-cb8b-11eb-9c36-144f45b26e1d.png)

Reply:
we don't use screenshots. always use copy pasted code snippets. screenshots are for amateurs.

Reply:
and I know it is hovered. but still try to copy paste it if possible.

Reply:
> i thought about it first but when i hovered on the constant i saw that it's already declaring the type
> 
> ```ts
> const avatarVariant: "filled" | "notFilled"
> ``` 
Actually as mentioned in the comment 

> TypeScript infers types of variables when there is no explicit information available

If you assigned a value `string` to a declared variable named `firstName` for example  you will find it is type is `string, `
if you tried to `assign a number` in the next line it will throw an error.